eth/v1alpha1/block.proto

```diff
@@ -0,0 +1,203 @@
+// Copyright 2019 Prysmatic Labs.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+syntax = "proto3";
+
+package ethereum.eth.v1alpha1;
+
+import "eth/v1alpha1/attestation.proto";
+
+option csharp_namespace = "Ethereum.Eth.v1alpha1";
+option go_package = "github.com/prysmaticlabs/ethereumapis/eth/v1alpha1;eth";
+option java_multiple_files = true;
+option java_outer_classname = "BlockProto";
+option java_package = "org.ethereum.eth.v1alpha1";
+option php_namespace = "Ethereum\\Eth\\v1alpha1";
+
+// The Ethereum 2.0 beacon block.
+message Block {
+    // Beacon chain slot that this block represents.
+    uint64 slot = 1; 
+
+    // 32 byte root of the parent block.
+    bytes parent_root = 2;
+
+    // 32 byte root of the resulting state after processing this block.
+    bytes state_root = 3;
+
+    // The block body itself.
+    BlockBody body = 4;
+
+    // 96 byte BLS signature from the validator that produced this block.
+    bytes signature = 5;
+}
+
+// The block body of an Ethereum 2.0 beacon block.
+message BlockBody {
+    // The validators RANDAO reveal 96 byte value.
+    bytes randao_reveal = 1;
+
+    // A reference to the Ethereum 1.x chain.
+    Eth1Data eth1_data = 2;
+
+    // 32 byte field of arbitrary data. This field may contain any data and
+    // is not used for anything other than a fun message.
+    bytes graffiti = 3; 
+
+    // The max proposer slashings allowed in a block is 16
```

Comment:
These numbers are subject to change, and would require us maintaining these comments relevant to the latest constants. I'd recommend removing that fixed number from here for now.

Reply:
@terencechain, any feedback?

Reply:
I agree with Raul, maybe we can add a commented note here to refer to the spec for limits.

Reply:
Yeah, I think im on board with that. It could be changed in a test configuration as well.